src/DonorDashboards/resources/js/app/components/select-control/index.js

```diff
@@ -11,7 +11,7 @@ import './style.scss';
 
 import {__} from '@wordpress/i18n';
 
-const SelectControl = ({label, value, isLoading, onChange, options, placeholder, width, isClearable}) => {
+const SelectControl = ({value, options, isLoading, label = null, onChange = null, placeholder = __('Select...', 'give'), width = null, isClearable = false}) => {
```

Comment:
Do we need a default parameter for value here?

Reply:
See https://github.com/impress-org/givewp/pull/7490#issuecomment-2310556555